src/PublicApiGenerator/ApiGenerator.cs

```diff
@@ -41,14 +41,30 @@ public static string GeneratePublicApi(Assembly assembly, Type[]? includeTypes =
                 {
                     return CreatePublicApiForAssembly(
                         asm,
-                        typeDefinition => includeTypes == null || includeTypes.Any(type => type.FullName == typeDefinition.FullName && type.Assembly.FullName == typeDefinition.Module.Assembly.FullName),
-                        shouldIncludeAssemblyAttributes,
-                        whitelistedNamespacePrefixes ?? defaultWhitelistedNamespacePrefixes,
+                        typeDefinition => options.IncludeTypes == null || options.IncludeTypes.Any(type => type.FullName == typeDefinition.FullName && type.Assembly.FullName == typeDefinition.Module.Assembly.FullName),
```

Comment:
For now I decided to step on this level and not float the option type into the underlying method calls

Reply:
It is OK.